### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by trodge**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
...
+* identityplayform: deprecated `google_identity_platform_project_default_config` resource. Use `google_identity_platform_config` resource instead ([#6293](https://github.com/hashicorp/terraform-provider-google-beta/pull/6293))
+
+FEATURES:
+* **New Data Source:** `google_certificate_manager_certificate_map` ([#6316](https://github.com/hashicorp/terraform-provider-google-beta/pull/6316))
+* **New Resource:** `google_artifact_registry_vpcsc_config` ([#6265](https://github.com/hashicorp/terraform-provider-google-beta/pull/6265))
+* **New Resource:** `google_dialogflow_cx_security_settings` ([#6300](https://github.com/hashicorp/terraform-provider-google-beta/pull/6300))
+* **New Resource:** `google_gke_backup_restore_plan` ([#6278](https://github.com/hashicorp/terraform-provider-google-beta/pull/6278))
+* **New Resource:** `google_scc_project_custom_module` ([#6315](https://github.com/hashicorp/terraform-provider-google-beta/pull/6315))
+* **New Resource:** `google_tpu_v2_vm` (beta) ([#6264](https://github.com/hashicorp/terraform-provider-google-beta/pull/6264))
```
Comment (melinath) [inline CHANGELOG.md:13]:
```suggestion
* **New Resource:** `google_tpu_v2_vm` ([#6264](https://github.com/hashicorp/terraform-provider-google-beta/pull/6264))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+* compute: updated the `autoscaling_policy.mode` to accept `ONLY_SCALE_OUT` on `google_compute_autoscaler` ([#6304](https://github.com/hashicorp/terraform-provider-google-beta/pull/6304))
+* compute: added `server_tls_policy` argument to `target_https_proxy` resource ([#6269](https://github.com/hashicorp/terraform-provider-google-beta/pull/6269))
+* compute: added `member` attribute to `google_compute_default_service_account` datasource ([#6311](https://github.com/hashicorp/terraform-provider-google-beta/pull/6311))
+* compute: added output field `internal_ipv6_prefix` to `google_compute_subnetwork` resource ([#6306](https://github.com/hashicorp/terraform-provider-google-beta/pull/6306))
+* container: added `node_config.fast_socket` field to `google_container_node_pool` ([#6289](https://github.com/hashicorp/terraform-provider-google-beta/pull/6289))
+* container: promoted `node_pool_auto_config` field in `google_container_cluster` from beta provider to GA provider. (ga) ([#6299](https://github.com/hashicorp/terraform-provider-google-beta/pull/6299))
```
Comment (melinath) [inline CHANGELOG.md:26]:
```suggestion
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+* container: promoted field `placement_policy.tpu_topology` in resource `google_container_node_pool` to GA ([#6286](https://github.com/hashicorp/terraform-provider-google-beta/pull/6286))
```
Comment (melinath) [inline CHANGELOG.md:27]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+* identityplatform: added `sign-in` field to `google_identity_platform_config` resource ([#6293](https://github.com/hashicorp/terraform-provider-google-beta/pull/6293))
+* networkconnectivity: added support for `linked_vpc_network` in `network_connectivity_spoke` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
+* networkservices: increased default timeout for `google_network_services_edge_cache_origin` to 120m from 60m ([#6275](https://github.com/hashicorp/terraform-provider-google-beta/pull/6275))
+* networkservices: increased default timeout for `google_network_services_edge_cache_service` to 60m from 30m ([#6281](https://github.com/hashicorp/terraform-provider-google-beta/pull/6281))
+* secretmanager: added `is_secret_data_base64` field to `google_secret_manager_secret_version` resource ([#6273](https://github.com/hashicorp/terraform-provider-google-beta/pull/6273))
+* workstations: added `env` field to `google_workstations_workstation` resource (beta) ([#6258](https://github.com/hashicorp/terraform-provider-google-beta/pull/6258))
```
Comment (melinath) [inline CHANGELOG.md:37]:
```suggestion
* workstations: added `env` field to `google_workstations_workstation` resource ([#6258](https://github.com/hashicorp/terraform-provider-google-beta/pull/6258))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,55 @@
...
+
+BUG FIXES:
+* bigquery: update documentation for `google_bigquery_table.time_partitioning.expiration_ms` ([#6290](https://github.com/hashicorp/terraform-provider-google-beta/pull/6290))
+* bigtable: added a read timeout to `google_bigtable_instance` ([#6276](https://github.com/hashicorp/terraform-provider-google-beta/pull/6276))
+* bigtable: improved regional reliability when instance overlaps a downed region in the resource `google_bigtable_instance` ([#6313](https://github.com/hashicorp/terraform-provider-google-beta/pull/6313))
+* eventarc: field `event_data_content_type` in `eventarc_trigger` now marked as computed ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```
Comment (melinath) [inline CHANGELOG.md:43]: this looks like a breaking change; just want to double-check it's fine in a minor release?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,55 @@
...
+* containeraws: added support for `auto_repair` in `container_aws_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
+* containerazure: added support for `auto_repair` in `container_azure_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
+* filestore: added support for the `"ZONAL"` value to `tier` in `google_filestore_instance` ([#6303](https://github.com/hashicorp/terraform-provider-google-beta/pull/6303))
+* firestore: added `delete_protection_state` field to `google_firestore_database` resource. ([#6295](https://github.com/hashicorp/terraform-provider-google-beta/pull/6295))
+* identityplatform: added `sign-in` field to `google_identity_platform_config` resource ([#6293](https://github.com/hashicorp/terraform-provider-google-beta/pull/6293))
+* networkconnectivity: added support for `linked_vpc_network` in `network_connectivity_spoke` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```
Comment (melinath) [inline CHANGELOG.md:33]:
```suggestion
* networkconnectivity: added support for `linked_vpc_network` in `google_network_connectivity_spoke` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```

File: CHANGELOG.md (hunk 5 above)
Comment (melinath) [reply CHANGELOG.md:43]:
```suggestion
* eventarc: field `event_data_content_type` in `google_eventarc_trigger` now marked as computed ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,55 @@
...
+* compute: added output field `internal_ipv6_prefix` to `google_compute_subnetwork` resource ([#6306](https://github.com/hashicorp/terraform-provider-google-beta/pull/6306))
+* container: added `node_config.fast_socket` field to `google_container_node_pool` ([#6289](https://github.com/hashicorp/terraform-provider-google-beta/pull/6289))
+* container: promoted `node_pool_auto_config` field in `google_container_cluster` from beta provider to GA provider. (ga) ([#6299](https://github.com/hashicorp/terraform-provider-google-beta/pull/6299))
+* container: promoted field `placement_policy.tpu_topology` in resource `google_container_node_pool` to GA ([#6286](https://github.com/hashicorp/terraform-provider-google-beta/pull/6286))
+* containeraws: added support for `auto_repair` in `container_aws_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
+* containerazure: added support for `auto_repair` in `container_azure_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```
Comment (melinath) [inline CHANGELOG.md:29]:
```suggestion
* containerazure: added support for `auto_repair` in `google_container_azure_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,55 @@
...
+* compute: added `member` attribute to `google_compute_default_service_account` datasource ([#6311](https://github.com/hashicorp/terraform-provider-google-beta/pull/6311))
+* compute: added output field `internal_ipv6_prefix` to `google_compute_subnetwork` resource ([#6306](https://github.com/hashicorp/terraform-provider-google-beta/pull/6306))
+* container: added `node_config.fast_socket` field to `google_container_node_pool` ([#6289](https://github.com/hashicorp/terraform-provider-google-beta/pull/6289))
+* container: promoted `node_pool_auto_config` field in `google_container_cluster` from beta provider to GA provider. (ga) ([#6299](https://github.com/hashicorp/terraform-provider-google-beta/pull/6299))
+* container: promoted field `placement_policy.tpu_topology` in resource `google_container_node_pool` to GA ([#6286](https://github.com/hashicorp/terraform-provider-google-beta/pull/6286))
+* containeraws: added support for `auto_repair` in `container_aws_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```
Comment (melinath) [inline CHANGELOG.md:28]:
```suggestion
* containeraws: added support for `auto_repair` in `google_container_aws_node_pool` ([#6282](https://github.com/hashicorp/terraform-provider-google-beta/pull/6282))
```

File: CHANGELOG.md [hunk 9]
```diff
@@ -1,5 +1,55 @@
...
+* **New Resource:** `google_edgenetwork_network` ([#6305](https://github.com/hashicorp/terraform-provider-google-beta/pull/6305))
+* **New Resource:** `google_edgenetwork_subnet` ([#6305](https://github.com/hashicorp/terraform-provider-google-beta/pull/6305))
+
+IMPROVEMENTS:
+* alloydb: added `network_config` field to support named IP ranges. ([#6297](https://github.com/hashicorp/terraform-provider-google-beta/pull/6297))
+* cloudrunv2: added fields `network_interfaces` to resource `google_cloud_run_v2_job` to support Direct VPC egress. ([#6287](https://github.com/hashicorp/terraform-provider-google-beta/pull/6287))
+* cloudrunv2: added fields `network_interfaces` to resource `google_cloud_run_v2_service` to support Direct VPC egress. ([#6287](https://github.com/hashicorp/terraform-provider-google-beta/pull/6287))
+* compute: updated the `autoscaling_policy.mode` to accept `ONLY_SCALE_OUT` on `google_compute_autoscaler` ([#6304](https://github.com/hashicorp/terraform-provider-google-beta/pull/6304))
+* compute: added `server_tls_policy` argument to `target_https_proxy` resource ([#6269](https://github.com/hashicorp/terraform-provider-google-beta/pull/6269))
```
Comment (melinath) [inline CHANGELOG.md:22]:
```suggestion
* compute: added `server_tls_policy` argument to `google_compute_target_https_proxy` resource ([#6269](https://github.com/hashicorp/terraform-provider-google-beta/pull/6269))
```

File: CHANGELOG.md [hunk 10]
```diff
@@ -1,5 +1,55 @@
...
+* **New Resource:** `google_artifact_registry_vpcsc_config` ([#6265](https://github.com/hashicorp/terraform-provider-google-beta/pull/6265))
+* **New Resource:** `google_dialogflow_cx_security_settings` ([#6300](https://github.com/hashicorp/terraform-provider-google-beta/pull/6300))
+* **New Resource:** `google_gke_backup_restore_plan` ([#6278](https://github.com/hashicorp/terraform-provider-google-beta/pull/6278))
+* **New Resource:** `google_scc_project_custom_module` ([#6315](https://github.com/hashicorp/terraform-provider-google-beta/pull/6315))
+* **New Resource:** `google_tpu_v2_vm` (beta) ([#6264](https://github.com/hashicorp/terraform-provider-google-beta/pull/6264))
+* **New Resource:** `google_edgenetwork_network` ([#6305](https://github.com/hashicorp/terraform-provider-google-beta/pull/6305))
+* **New Resource:** `google_edgenetwork_subnet` ([#6305](https://github.com/hashicorp/terraform-provider-google-beta/pull/6305))
+
+IMPROVEMENTS:
+* alloydb: added `network_config` field to support named IP ranges. ([#6297](https://github.com/hashicorp/terraform-provider-google-beta/pull/6297))
```
Comment (melinath) [inline CHANGELOG.md:18]:
```suggestion
* alloydb: added `network_config` field to support named IP ranges on `google_alloydb_cluster` ([#6297](https://github.com/hashicorp/terraform-provider-google-beta/pull/6297))
```

File: CHANGELOG.md [hunk 11]
```diff
@@ -1,5 +1,55 @@
 ## 4.84.0 (Unreleased)
 
+DEPRECATIONS:
+* alloydb: deprecated `network` field in favor of `network_config`. ([#6297](https://github.com/hashicorp/terraform-provider-google-beta/pull/6297))
```
Comment (melinath) [inline CHANGELOG.md:4]:
```suggestion
* alloydb: deprecated `network` field in favor of `network_config` on `google_alloydb_cluster`. ([#6297](https://github.com/hashicorp/terraform-provider-google-beta/pull/6297))
```

**Review by melinath — CHANGES_REQUESTED** [review]
double-checking you heard about https://github.com/GoogleCloudPlatform/magic-modules/pull/8876#issuecomment-1721686745 as well?

File: CHANGELOG.md (hunk 5 above)
Comment (trodge) [reply CHANGELOG.md:43]: I believe the field went from optional to optional+computed.

**Review by melinath — APPROVED** [review]

**Pull request merged**